pipeline/src/scoring/score_parameters.jl

```diff
@@ -0,0 +1,45 @@
+"""
+Internal function for making a DataFrame for a given parameter using the
+    provided MCMC samples and the truth value.
+"""
+function _make_prediction_dataframe(param_name, samples, truth; model = "EpiAware")
+    x = samples[Symbol(param_name)][:]
+    DataFrame(predicted = x, observed = truth, model = model,
+        parameter = param_name, sample_id = 1:length(x))
+end
+
+"""
+Internal function for scoring a DataFrame containing a prediction and truth value
+    for a parameter using the `scoringutils` package.
+"""
+function _score(df)
+    @rput df
+    R"""
+    library(scoringutils)
+    result = df |> as_forecast() |> score() |> summarize_scores(by = "parameter")
```

Comment:
it seems weird to be summarising by score and hard coding this -> given we know we want to be breaking this down by other strata (in the analysis plan). Why not just get rid of the summarise_scores call and summarise in julia as we need?

Reply:
or passing scores out to summarise_scores (its quite handy as it knows about scores to summarise).